tests/phpunit/tests/media.php

```diff
@@ -3973,6 +3973,54 @@ public function test_wp_generate_attachment_metadata_doesnt_generate_sizes_for_1
 		);
 	}
 
+	/**
+	 * @ticket 58212
+	 * @covers ::wp_get_attachment_image()
+	 */
+	public function test_wp_get_attachment_image_context_filter_default() {
+		$last_context = '';
+		add_filter(
+			'wp_get_attachment_image_context',
+			function( $context ) use ( &$last_context ) {
```

Comment:
Since `$this` is not used, this will [reduce memory](https://stackoverflow.com/a/64481894/93579) (marginally):

```suggestion
			static function( $context ) use ( &$last_context ) {
```